security/pkg/nodeagent/caclient/providers/citadel/client.go

```diff
@@ -102,102 +132,19 @@ func (c *citadelClient) CSRSign(ctx context.Context, reqID string, csrPEM []byte
 		return nil, err
 	}
 
-	if len(resp.CertChain) <= 1 {
```

Comment:
Why is this removed?

Reply:
I checked it in Signing Client before return to citadel